index.js

```diff
@@ -1,7 +1,7 @@
 import express from 'express';
 import bodyParser from 'body-parser';
 import dotenv from 'dotenv';
-import routes from './routes/index';
+// import routes from './routes/index';
```

Comment:
I'm not sure it's the best idea to have commented code in a PR. In any case, is this unnecessary? Won't it still come up when we write routes? If it will then it would be best to leave it as is till we use it.

Reply:
Once again, thanks for the feedback and pointing this out @LukasChiama . I would have deleted the line entirely, but I opted to comment it because I want the team to know that we are still going to use the `routes/` directory in our subsequent route handlers.

I gave @tolulope-od a lengthy answer to the reason why I "muted" the `routes/` directory temporarily. I hope you wouldn't mind to check that up please?